dask_jobqueue/pbs.py

```diff
@@ -95,6 +95,10 @@ def __init__(self,
 
         logger.debug("Job script: \n %s" % self.job_script())
 
+    def _jobid_from_submit_output(self, out):
```

Comment:
I don't think this method is meant to be used directly, so it would be good to hide it so it doesn't show up in available methods for people. and there a several private markers in core.py.

Reply:
No strong feeling about the leading underscore, but I would put an underscore between job and id though i.e. `_job_id_from_submit_output`.

Personally, given that the function is one-line long and that the name of the function is reasonably self-explanatory, I would not bother documenting it.